src/dashboards/components/AutoRefresh.scss

```diff
@@ -0,0 +1,46 @@
+.refresh-form-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 10px 50px;
+}
+
+.refresh-form-container-child {
+  flex: 1;
+  &.active {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
```

Comment:
TIL

Reply:
Freaking love CSS Grid and Flex... Gd bless whoever made them